tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+envlist =
+    py27
+    flake8
+    pylint
+
+[testenv]
+deps =
+    coverage
+    mock
+commands =
+    coverage run --source=tests -m unittest discover tests
```

Comment:
This only runs the coverage on `tests` subdirectory (so it will be always 100%), not the whole repository. You can use `--source=.` to fix that.